doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -607,344 +797,50 @@ Response code:
 
 + Response 204
 
+## Standard queryContext [/v2/queryContext]
 
-## Context Entity Subscriptions [/v2/entities/{entityId}/subscriptions]
-
-### List all subscriptions present in the system [GET /v2/subscriptions]
-
-+ Response 200
-
-        [
-            {
-                "id": "abcdefg",
-                "entity": {
-                    "id": "Bcn_Welt",
-                    "type": "Room"
-                },
-                "attributes": [
-                    "temperature"
-                ],
-                "callback": "http://localhost:1234",
-                "options": {
-                    "duration": "1M",
-                    "condition": "ONCHANGE",
-                    "throttling": "PT5S"
-                }
-            },
-             {
-                "id": "wyztrew",
-                "entity": {
-                    "pattern": "DC_*",
-                    "type": "Room"
-                },
-                "attributes": [
-                    "temperature"
-                ],
-                "callback": "http://localhost:1234"
-            }
-        ]
-
-### Get the details of a subscription [GET /v2/subscriptions/{subscriptionId}]
-
-+ Parameters
-    + subscriptionId: abcdefg (required, string) - ID of the suscription
-
-+ Response 200
-
-        {
-            "id": "abcdefg",
-            "entity": {
-                "id": "Bcn_Welt",
-                "type": "Room"
-            },
-            "attributes": [
-                "temperature"
-            ],
-            "callback": "http://localhost:1234",
-            "options": {
-                "duration": "1M",
-                "condition": "ONCHANGE",
-                "throttling": "PT5S"
-            }
-        }
-
-### Remove a suscription [DELETE /v2/subscriptions/{subscriptionId}]
-
-+ Parameters
-    + subscriptionId: abcdefg (required, string) - ID of the suscription
-
-+ Response 204
-
-
-### Create a new subscription [POST /v2/subscriptions]
+### queryContext [POST /v2/queryContest]
 
-+ Request (application/json)
+## Standard updateContext [/v2/updateContext]
```

Comment:
I think standard operations are already described by v1 apiary document. So I think a link between the 2 documents might suffice